datajoint/autopopulate.py

```diff
@@ -222,7 +222,11 @@ def handler(signum, frame):
             make_kwargs=make_kwargs,
         )
 
-        if processes == 1:
+        if processes < 0:
+            raise Exception("processes must not be negative")
+        elif processes == 0:
+            return error_list, nkeys
```

Comment:
here `error_list` is empty.  Why does this need to be return? What's the use case for `processes == 0`?

Reply:
I think if `nkeys <= 0`, it should just quietly exit. That's consistent with previous behavior.